nautobot/docs/development/apps/api/views/help-documentation.md

```diff
@@ -0,0 +1,13 @@
+# Adding Help Documentation
```

Comment:
This file is part of what we'll have to rework in terms of making model documentation easier to link from the web UI.